src/main/java/com/google/sps/servlets/ErrorServlet.java

```diff
@@ -55,9 +64,10 @@ private void processError(HttpServletRequest request,
     if (DataUtils.isEmptyParameter(requestUri)) {
       requestUri = "Unknown";
     }
-    response.setContentType("text/html");
 
     response.getWriter().println("<h1>" + statusCode + "</h1>"
-                                 + "<p>" + throwable.getMessage() + "</p>");
+                                 + "<p>" + throwable.getMessage() + "</p>" + "</ br>"
+                                 + "<p>" + "on" servletName + "</p>" + "</ br>"
+                                 + "<p>" + "when accessing" requestUri + "</p>");
```

Comment:
I think you need a `+` before `requestUri`.

Also, a space before the closing quote, both on this line and the previous one.

Also, the two instances of `</ br>` don't seem to do anything (no matching tag, and redundant with `<p>` anyways).